src/routes/index.ts

```diff
@@ -1,12 +1,14 @@
 import {Router} from 'express'
 import {healthRouter} from './health'
 import {helloRouter} from './hello'
+import spentRouter from './spent'
 
 const router = Router()
 
 // Wire up routers
 router.use('/', helloRouter)
 router.use('/health', healthRouter)
+router.use('/spent', spentRouter)
```

Comment:
Seria legal seguir as boas práticas do padrão REST. Eu também renomearia o router. Vou deixar uma sugestão:

```suggestion
router.use('/expenses', expensesRouter)
```